code/game/machinery/cryopod.dm

```diff
@@ -192,12 +192,13 @@
 		/obj/item/device/paicard,
 		/obj/item/weapon/gun,
 		/obj/item/weapon/pinpointer,
-		/obj/item/clothing/suit,
```

Comment:
@FlattestGuitar You're right that the RD armour and sec armour gets missed. I'm re-upping now. Also, I'm adding both of the CMO's items, the krav maga gloves, jetpacks, captains medal and plasma tanks... which all have been missed by the original system as well.